pepper-apis/ddp-common/src/main/java/org/broadinstitute/ddp/db/TransactionWrapper.java

```diff
@@ -390,12 +390,10 @@ private static String stackTraceToString(StackTraceElement[] stackTrace) {
      * @param <X>      Type of exception thrown by callback
      * @param callback Accepts a handle  @throws X exception thrown by callback
      * @throws DDPException if error getting a raw connection
-     * @deprecated Please use JDBI instead of JDBC via {@link #useTxn(HandleConsumer)} or
      * {@link #withTxn(HandleCallback)}
      * <p>Use a jdbc connection within a transaction.  If more than one database has been
      * initialized, an exception is thrown.</p>
      */
-    @Deprecated
```

Comment:
This change is long overdue. We have no intention of changing our use of this technique, so the deprecation tag was just gumming things up.